lib/kitchen/directions/move_solutions_to_answer_key/v1.rb

```diff
@@ -11,6 +11,8 @@ def bake(chapter:, metadata_source:, strategy:, append_to:)
           Strategies::UPhysics
         when :american_government
           Strategies::AmericanGovernment
```

Comment:
this may be why codecov is complaining ... does this strategy exist anymore? if not, this case is not needed